llvm/lib/Transforms/Vectorize/LoopVectorizationPlanner.h

```diff
@@ -276,6 +276,20 @@ class VPBuilder {
     return tryInsertInstruction(new VPPhi(IncomingValues, DL, Name));
   }
 
+  VPValue *createElementCount(Type *Ty, ElementCount EC) {
```

Comment:
```suggestion
  VPValue *createRuntimeVF(Type *Ty, ElementCount VF) {
```
Sounds a bit odd to createElementCount given an ElementCount. There's probably a better name than "RuntimeVF", but that's already in use elsewhere, in any case should be clearly defined.

Reply:
IRBuilder has an identical function also called `IRBuilderBase::CreateElementCount`, and SelectionDAG too with `SelectionDAG::getElementCount`. I think we should be consistent here.

It also doesn't necessarily need to generate a VF. An ElementCount could be e.g. VFxUF or another arbitrary quantity.